### ISSUE TITLE
The type `System.Collections.Generic.IEnumerable<StarWars.Models.ICharacter>` of field `Query.character` could not be resolved to a valid schema type

### ISSUE THREAD

**Issue by ThunderDev1** [ChilliCream/hotchocolate#260]
**Describe the bug**

Starwars template project throws the following error, for any query:

```The type `System.Collections.Generic.IEnumerable<StarWars.Models.ICharacter>` of field `Query.character` could not be resolved to a valid schema type```

**To Reproduce**
1. dotnet new starwars
2. dotnet run --project StarWars/StarWars.csproj -c release
3. Open GraphiQL and execute any query on http://localhost:5000

**Expected behavior**
The query executes correctly and returns a result

**Desktop:**
 - OS: Windows 10
 - dotnet core 2.1

**Comment by michaelstaib** [comment]
I will have a look at it.

**Comment by michaelstaib** [comment]
hi ThunderDev1,

I can confirm the bug. I will fix it and redeploy the template.

**Comment by michaelstaib** [comment]
Ok, the issue is fixed and release with the template version 0.4.6.

[Release Info](https://github.com/ChilliCream/hotchocolate-templates/releases/tag/0.4.6)

It will take a few minutes for nuget to update its package cache. 

[Nuget](https://www.nuget.org/packages/HotChocolate.Templates.StarWars/0.4.6)

You can update your nuget template like this:

```bash
dotnet new -u HotChocolate.Templates.StarWars
dotnet new -i HotChocolate.Templates.StarWars
``` 

This will uninstall the old version and install the new. 
 

Thank you for reporting this issue.  👍
